src/openrct2/interface/Window.cpp

```diff
@@ -347,6 +348,7 @@ void WindowCloseTop()
 void WindowCloseAll()
 {
     WindowCloseByClass(WindowClass::Dropdown);
+    WindowCloseByClass(WindowClass::Loadsave);
```

Comment:
The code below very specifically excluded windows that are stuck to the back or the front, of which the Load/Save window is one. Since that it done intentionally, you shouldn’t just add an exemption.